physcraper/__init__.py

```diff
@@ -1809,186 +1810,168 @@ def run_blast_wrapper(self):  # TODO Should this be happening elsewhere?
                                                                                                             last_blast))
         self._blasted = 1
 
-    def get_acc_from_blast(self, query_string):
+    def get_new_seqs_for_mergedseq(self, var_list, query_dict):
```

Comment:
This is an improvement! But it still has a pretty hard to follow the long series of if-else's. Is it possible to separate out the different pieces into functions, rather than all together in one? e.g. a function that takes each  staxids_l (I'm still not clear on what the staxids_l has in in it...) and returns the info you need for it? Or is there another place you can find that it would be possible to separate this into more pieces?